### PR TITLE
Update Injector, Add e2e tests, cleanup

### PR DESCRIPTION
**Pull request by edtoken**
- Update Injector, Add all elements in withFramePay HOC
- Add multiple payment methods example with new withFramePay HOC
- Update tests, Add new e2e tests
- Cleanup react-redux example

## Update
- Delete repo examples
- Update constants, Add default CDN FramePay links
- Update Provider Add remote error details
- Update context/Injector fix type declarations problem, Add FramePay types export

### CODE REVIEW

File: test/e2e/specs/bank-separate.spec.ts [hunk 1]
```diff
@@ -14,30 +14,21 @@ describe('bank-separate', () => {
...
     });
 
-    it('should be call the on-ready card element hook', async () => {
-        // @ts-ignore
-        const isReady = await page.getAttributeOf(
-            '#events-bankAccountType-onReady-true',
-            'data-value'
-        );
-        expect(isReady).toEqual('true');
+    it('should be call the on-ready BankAccountType element hook', async () => {
```
Comment (adamaltman) [inline test/e2e/specs/bank-separate.spec.ts]:
```suggestion
    it('should call the on-ready BankAccountType element hook', async () => {
```

File: test/e2e/specs/bank-separate.spec.ts [hunk 2]
```diff
@@ -14,30 +14,21 @@ describe('bank-separate', () => {
...
     });
 
-    it('should be call the on-ready cvv element hook', async () => {
-        // @ts-ignore
-        const isReady = await page.getAttributeOf(
-            '#events-bankAccountNumber-onReady-true',
-            'data-value'
-        );
-        expect(isReady).toEqual('true');
+    it('should be call the on-ready BankAccountNumber element hook', async () => {
```
Comment (adamaltman) [inline test/e2e/specs/bank-separate.spec.ts]:
```suggestion
    it('should call the on-ready BankAccountNumber element hook', async () => {
```

File: test/e2e/specs/bank-separate.spec.ts [hunk 3]
```diff
@@ -14,30 +14,21 @@ describe('bank-separate', () => {
...
     });
 
-    it('should be call the on-ready expiry element hook', async () => {
-        // @ts-ignore
-        const isReady = await page.getAttributeOf(
-            '#events-bankRoutingNumber-onReady-true',
-            'data-value'
-        );
-        expect(isReady).toEqual('true');
+    it('should be call the on-ready BankRoutingNumber element hook', async () => {
```
Comment (adamaltman) [inline test/e2e/specs/bank-separate.spec.ts]:
```suggestion
    it('should call the on-ready BankRoutingNumber element hook', async () => {
```

File: test/e2e/specs/multiple-methods.spec.ts [hunk 4]
```diff
@@ -0,0 +1,263 @@
+describe('bank-separate', () => {
+    beforeAll(async () => {
+        await page.goto(`${location}/multiple-methods`);
+    });
+
+    it('should load the page', async () => {
+        await expect(page.title()).resolves.toMatch('Test Multiple Methods');
+    });
+
+    it('should be call the on-ready card element hook', async () => {
```
Comment (adamaltman) [inline test/e2e/specs/multiple-methods.spec.ts]:
```suggestion
    it('should call the on-ready card element hook', async () => {
```

File: README.md [hunk 5]
```diff
@@ -26,14 +27,14 @@ This package is a wrapper for FramePay offering out-of-the-box support for Redux
...
-- [Use with TypeScript](example/react-typescript)   
-- [Examples from README file](example/readme-example)
-- [Multiple Payment Methods](example/react-router/src/elements/MultiplePaymentMethods.js)      
-- [Separated fields](example/react-router/src/elements/PaymentCardSeparated.js)      
-- [Other Payment Methods](example/react-router/src/elements/OtherPaymentMethods.js)      
+- [Use with react-redux (codesandbox)](https://codesandbox.io/s/n089y731x4)    
+- [Use with react-router (codesandbox)](https://codesandbox.io/s/z2q2lx9ry4)      
+- [Examples from README file (codesandbox)](https://codesandbox.io/s/8ly7ml77y9)
+- [Multiple Payment Methods (codesandbox)](https://codesandbox.io/s/z2q2lx9ry4?module=/src/elements/MultiplePaymentMethods.js)      
+- [Multiple Payment Methods Short version (codesandbox)](https://codesandbox.io/s/z2q2lx9ry4?module=/src/elements/MultiplePaymentMethodsShort.js)
```
Comment (adamaltman) [inline README.md]: I get this error for example:

DependencyNotFoundError
Could not find dependency: 'framepay-react' relative to '/src/App.js'

Comment (edtoken) [reply README.md]: @adamaltman it doesn't work until the `framepay-react` doesn't released.
Right after the release i check and fix if necessary

**Pull request merged**